templates/items/details/details-equipment.hbs

```diff
@@ -26,7 +26,7 @@
             {{!-- Max Dex --}}
             {{#if hasDexModifier}}
             {{ formField fields.armor.fields.dex value=source.armor.dex label="DND5E.ItemEquipmentDexModAbbr"
-                         placeholder="∞" localize=true classes="label-top" }}
+                         placeholder=(ifThen (and (gte (lookup config.armorMaxDex source.type.value) 0) (ne (lookup config.armorMaxDex source.type.value) null)) (dnd5e-numberFormat (lookup config.armorMaxDex source.type.value)) "∞") localize=true classes="label-top" }}
```

Comment:
Should probably be in the `getData` method instead given how verbose this one line is.